src/qt/utils.cpp

```diff
@@ -29,7 +29,7 @@
 #include <QtCore>
 #include <QApplication>
 
-#include "src/qt/TailsOS.h"
+#include "qt/TailsOS.h"
```

Comment:
shouldn't this be just "TailsOS.h" like the "utils.h" below?

Reply:
Yep, thanks, fixed.